models/dish/dishModel.js

```diff
@@ -0,0 +1,77 @@
+import { Schema, model } from 'mongoose';
+
+const dishSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: [true, 'Name of the dish is required'],
+      unique: true,
```

Comment:
i'm not sure it should be unique, "mashed potato" can be introduced by many chefs, for example

Reply:
Sure